testbeam_analysis/dut_alignment.py

```diff
@@ -832,18 +813,21 @@ def alignment(input_track_candidates_file, input_alignment_file, n_pixels, pixel
         does not have to be used in the fit itself! This is useful for time reference planes.
         E.g.  To use telescope planes (first and last 3 planes) + time reference plane (3)
         selection_hit_duts = [0, 1, 2, 4, 5, 6, 7]
-    selection_track_quality : uint or iterable or iterable of iterable
+    quality_sigma : float
         Track quality for each hit DUT.
     initial_rotation : array
-        Initial rotation array.
+        Initial rotation array. If None, deduce the rotation from pre-alignment.
     initial_translation : array
-        Initial translation array.
+        Initial translation array. If None, deduce the translation from pre-alignment.
     max_iterations : uint
         Maximum number of iterations of calc residuals, apply rotation refit loop until constant result is expected.
         Usually the procedure converges rather fast (< 5 iterations)
-    use_n_tracks: uint
-        Defines the amount of tracks to be used for the alignment. More tracks can potentially make the result
-        more precise, but will also increase the calculation time.
+    max_events: uint
+        Radomly select max_events for alignment. If None, use all events, which might slow down the alignment.
+    use_fit_limits : bool
```

Comment:
Really needed to be a parameter? Either it enhances the analysis and should be always true, or feature can be omitted to simplify function call.